tools/curl_post/curl_post.xml

```diff
@@ -1,17 +1,24 @@
 <?xml version="1.0"?>
-<tool name="cURL POST" id="curl_post" version="0.0.1">
+<tool name="cURL POST" id="curl_post" version="0.0.2">
```

Comment:
Why is this tool needed at all? Is the upload tool not good enough?

Reply:
@bgruening This tool is intended for uploading results to a generic external server. As far as I know, the upload tool is for downloading data only.